app/controllers/collection_templates_controller.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+class CollectionTemplatesController < CollectionDraftsController
+  include CollectionsCmrHelper
+  before_action :templates_enabled?
+  before_action :set_resource, only: %i[create_draft destroy edit update show]
+  before_action :load_umm_schema, only: %i[new edit show new_from_existing]
+
+  def create_draft
+    authorize get_resource
+    draft = CollectionDraft.create_from_template(get_resource, current_user)
+    Rails.logger.info("Audit Log: Collection Draft #{draft.entry_title} was created from a template by #{current_user.urs_uid} in provider: #{current_user.provider_id}")
```

Comment:
I suspect it was chosen because it mirrors the logging in collection drafts, but I will try to craft a more useful message to log when I update that feature in MMT-38.